src/main/kotlin/Application.kt

```diff
@@ -0,0 +1,37 @@
+import com.hexagonkt.helpers.require
+import com.hexagonkt.http.server.Server
+import com.hexagonkt.http.server.ServerPort
+import com.hexagonkt.http.server.jetty.JettyServletAdapter
+import com.hexagonkt.injection.InjectionManager
+import com.hexagonkt.settings.SettingsManager.settings
+import com.hexagonkt.store.Store
+import com.hexagonkt.store.mongodb.MongoDbStore
+import models.Message
+import models.User
+import routes.router
+
+internal val injector = InjectionManager.apply {
+    bindObject<ServerPort>(JettyServletAdapter())
+    bindObject(User::class, createUserStore())
+    bindObject(Message::class, createMessageStore())
+}
+
+internal val server = Server(injector.inject(), router)
```

Comment:
I guess you can skip `injector.inject()`. Isn't feasible to use  `Server(router)`?